### PR TITLE
added changes described.

### PR DESCRIPTION
**Pull request by Malind19**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md [hunk 1]
```diff
@@ -29,12 +29,14 @@ Enterprises can use [Microsoft BitLocker Administration and Monitoring (MBAM)](h
...
+Devices joined to Azure AD are managed using Mobile Device Management (MDM) policy from an MDM solution such as Microsoft Intune. Without Windows 10 v1809, only local administrators can enable Bitlocker via Intune policy. [BitLocker Device Encryption](bitlocker-device-encryption-overview-windows-10.md#bitlocker-device-encryption) status can be queried from managed machines via the [Policy Configuration Settings Provider (CSP)](https://docs.microsoft.com/windows/client-management/mdm/policy-configuration-service-provider), which reports on whether BitLocker Device Encryption is enabled on the device. Compliance with BitLocker Device Encryption policy can be a requirement for [Conditional Access](https://www.microsoft.com/cloud-platform/conditional-access) to services like Exchange Online and SharePoint Online.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
Devices joined to Azure AD are managed using Mobile Device Management (MDM) policy from an MDM solution such as Microsoft Intune. Without Windows 10 v1809, only local administrators can enable BitLocker via Intune policy. [BitLocker Device Encryption](bitlocker-device-encryption-overview-windows-10.md#bitlocker-device-encryption) status can be queried from managed machines via the [Policy Configuration Settings Provider (CSP)](https://docs.microsoft.com/windows/client-management/mdm/policy-configuration-service-provider), which reports on whether BitLocker Device Encryption is enabled on the device. Compliance with BitLocker Device Encryption policy can be a requirement for [Conditional Access](https://www.microsoft.com/cloud-platform/conditional-access) to services like Exchange Online and SharePoint Online.
```

Comment (nenonix) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
Devices joined to Azure AD are managed using Mobile Device Management (MDM) policy from an MDM solution such as Microsoft Intune. Without Windows 10 v1809, only local administrators can enable Bitlocker via Intune policy. [BitLocker Device Encryption](bitlocker-device-encryption-overview-windows-10.md#bitlocker-device-encryption) status can be queried from managed machines via the [Policy Configuration Settings Provider (CSP)](https://docs.microsoft.com/windows/client-management/mdm/policy-configuration-service-provider), which reports whether BitLocker Device Encryption is enabled on the device. Compliance with BitLocker Device Encryption policy can be a requirement for [Conditional Access](https://www.microsoft.com/cloud-platform/conditional-access) to services like Exchange Online and SharePoint Online.
```

File: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md [hunk 2]
```diff
@@ -29,12 +29,14 @@ Enterprises can use [Microsoft BitLocker Administration and Monitoring (MBAM)](h
...
 
 For hardware that is compliant with Modern Standby and HSTI, when using either of these features, [BitLocker Device Encryption](bitlocker-device-encryption-overview-windows-10.md#bitlocker-device-encryption) is automatically turned on whenever the user joins a device to Azure AD. Azure AD provides a portal where recovery keys are also backed up, so users can retrieve their own recovery key for self-service, if required. For older devices that are not yet encrypted, beginning with Windows 10 version 1703 (the Windows 10 Creators Update), admins can use the [BitLocker CSP](https://docs.microsoft.com/windows/client-management/mdm/bitlocker-csp) to trigger encryption and store the recovery key in Azure AD.
 
+This is applicable for Azure Hybrid AD as well.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
This is applicable to Azure Hybrid AD as well. 
```

File: windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md [hunk 3]
```diff
@@ -29,12 +29,14 @@ Enterprises can use [Microsoft BitLocker Administration and Monitoring (MBAM)](h
...
 
-Starting with Windows 10 version 1703 (also known as the Windows Creators Update), the enablement of BitLocker can be triggered over MDM either by the [Policy CSP](https://docs.microsoft.com/windows/client-management/mdm/policy-configuration-service-provider) or the [BitLocker CSP](https://docs.microsoft.com/windows/client-management/mdm/bitlocker-csp). The BitLocker CSP adds policy options that go beyond ensuring that encryption has occurred, and is available on computers that run Windows 10 Business or Enterprise editions and on Windows Phones.
+Starting with Windows 10 version 1703 (also known as the Windows Creators Update), the enablement of BitLocker can be triggered over MDM either by the [Policy CSP](https://docs.microsoft.com/windows/client-management/mdm/policy-configuration-service-provider) or the [BitLocker CSP](https://docs.microsoft.com/windows/client-management/mdm/bitlocker-csp). The BitLocker CSP adds policy options that go beyond ensuring that encryption has occurred, and is available on computers that run Windows 10 and on Windows Phones.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-management-for-enterprises.md]:
```suggestion
Starting with Windows 10 version 1703 (also known as the Windows Creators Update), the enablement of BitLocker can be triggered over MDM either by the [Policy CSP](https://docs.microsoft.com/windows/client-management/mdm/policy-configuration-service-provider) or the [BitLocker CSP](https://docs.microsoft.com/windows/client-management/mdm/bitlocker-csp). The BitLocker CSP adds policy options that go beyond ensuring that encryption has occurred, and is available on computers that run Windows 10 and on Windows phones.
```

**Comment by mypil** [comment]
@Malind19  - please merge the commit on the copy/edit review made by JohanFreelancer9 and nenonix or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by Malind19** [comment]
@mypil All possible commits are merged.

**Comment by mypil** [comment]
@Justinha @brianlic-msft - Can you please assist to check, approve and merge the content for this PR based on issue #2590?

Thank you.

**Comment by mypil** [comment]
@Malind19 - please apply the recommended changes from Justinha. Thank you

**Comment by Malind19** [comment]
@mypil Done

**Comment by mypil** [comment]
@Justinha  -Kindly check, approve and merge the content for this PR based on issue #2590?

Thank you.

**Comment by lizap** [comment]
I updated the version references to follow the standard naming ("Windows 10, version 1809" not "Windows 10 v1809"). Other than that, this is good, and I'm merging it. Thanks!

**Pull request merged**